### DISCUSSION TITLE
Wrong box characters rendering in code on Android devices

### DISCUSSION THREAD

**Discussion by thanhph111** [community/community#5599] [Mobile] [answered]
This issue has been around for quite some time but I didn't see any relevant discussion coming up so I decided to make one.

As you can see in the screenshots below, only _Android_ devices have display problems when drawing boxes, including code in markdown. It's also wrapped even when that option has been disabled (line 89, 91, 93, 95).

| _iOS_ screenshot | _Android_ screenshot |
| --- | --- |
| ![_iOS_ screenshot](https://user-images.githubusercontent.com/60842560/132952958-220af1f4-c159-498a-89da-5c2dde03c70b.png) | ![Screenshot_20210911-211054_GitHub](https://user-images.githubusercontent.com/60842560/132952997-e6d2aa51-a3e7-4acf-bbe0-2b6e3ea934dc.jpg) |
 
I think this might be related to the fonts?!

**Comment by aconsuegra** [comment, marked as answer]
Hi @thanhph111. Thanks for the report! 
Does this file belong to a public repository? If you could share the link with us, it would help us to debug the issue. In addition, I see that you have different values for the line wrapping setting on iOS and Android. Could you disable line wrapping on Android and double check if the issue persists? (if you are viewing a file, like in your screenshot, you can do that by tapping on the settings wheel in the toolbar).

**Reply by thanhph111** [reply]
Thanks for getting back to me! 

As I mentioned above, the wrap option is disabled, you can see the lines **86**, **87**, **88** in the image, they're not fully displayed. However, the lines **89**, **91**, **93**, **95** are not only fully visible but also wrapped.

I has record a screencast to demonstrate it more clearly with [the repository](https://github.com/thanhph111/box-drawing-test) I prepared. 
https://user-images.githubusercontent.com/60842560/133262786-e63b2890-f579-4a58-9a8c-8866cbaeee5d.mp4

I tested on the _SAMSUNG Galaxy Note FE_ (**SM-N935F/DS**) and the _SAMSUNG Galaxy J7 Prime_ (**SM-G610F/DS**).

Let me know if you need any further information.

**Reply by aconsuegra** [reply]
Thanks for putting together that repository! I can reproduce this on my end now. I will create an issue for our engineering team, and hopefully we can have a fix soon.
